packages/sdk/src/cosmwasmclient.ts

```diff
@@ -124,9 +132,10 @@ export class CosmWasmClient {
     return this.signingData.signCallback;
   }
 
-  private constructor(url: string, signingData?: SigningData) {
+  private constructor(url: string, signingData?: SigningData, feeTable?: Partial<FeeTable>) {
```

Comment:
You can make the `feeTable` non optional here. Since this constructor is private, only this file cares. Than the assignment expression below gets a little less crazy.